Info-Orbs/src/core/wifiWidget.cpp

```diff
@@ -1,10 +1,15 @@
 #include "core/wifiWidget.h"
 #include <WiFi.h>
+#include <WiFiManager.h> // https://github.com/tzapu/WiFiManager
+
 
 WifiWidget::WifiWidget(ScreenManager& manager) : Widget(manager) {}
 
 WifiWidget::~WifiWidget() {}
 
+// WiFiManager, Local intialization. Once its business is done, there is no need to keep it around
```

Comment:
but you keep it around indefinitely?

Reply:
This comment was taken from example code and I probably should have deleted it. In most cases the WiFiManager object can be discarded. But I plan to keep it running so the config portal is always available, so the user can browse to it and change Info-Obs settings like enabling or disabling widgets, changing styles and colors, etc.